.github/workflows/publish-to-dockerhub.yml

```diff
@@ -3,8 +3,8 @@ name: publish to DockerHub
 on:
   workflow_dispatch:
     inputs:
-      tag:
-        description: "Tag to publish under, defaults to latest"
+      tags:
+        description: "Comma-separated list of tags to publish (e.g., '0.2,latest')"
```

Comment:
Should we add a warning, to not use spaces here? Assuming, they would cause issues in `parse-tags`. ^^

Reply:
good point, added the space trimming